src/module/setup/utility-classes.js

```diff
@@ -577,6 +577,38 @@ export class MacroUtils {
     }
     return res;
   }
+
+  /**
+   * Select all allies - approximated by all linked actors in combat
+   * If selfUuid is set it excludes the specified actor, otherwise it includes all linked tokens
+   */
+  static getAllies(selfUuid="") {
+    let res = [];
+    if (!game.combat) return res;
+    const combatants = [...game.combat.combatants.values()];
+    combatants.forEach(c => {
+      if (c.token.isLinked && c.token.actor.uuid != selfUuid) res.push(c.token);
```

Comment:
I would also tie this into the token disposition. The value is at `c.token.disposition`, and the constants the values use are at `CONST.TOKEN_DISPOSITIONS`

Reply:
Good idea. Done.